socketshark/config_defaults.py

```diff
@@ -66,6 +66,10 @@
     'ping_timeout': 5,
 }
 
+# Alternative Redis configuration. Useful for redundancy or migration.
```

Comment:
I don't think the "redundancy" part is accurate if any one Redis not being available will make the service shut down.

Reply:
Thanks, I removed this in a415332.